src/v3/src/components/IdentifierContainer/IdentifierContainer.tsx

```diff
@@ -23,7 +23,6 @@ const IdentifierContainer: UISchemaElementComponent<{
   const { options: { identifier } } = uischema;
 
   const mainContainerClasses = classNames('identifier-container');
-  const identifierSpanClasses = classNames('identifier', 'no-translate');
```

Comment:
We've been having ongoing discussions with Odyssey about how we can move `no-translate` to instead use the `translate="no"` global HTML attribute (this is what I talked to you about for the checkbox component, @glenfannin-okta). I have an open PR to get this merged into Odyssey right now, which would unblock this for the identifier container. Maybe we should wait until that gets merged and then we can bump to latest version of Odyssey

Reply:
Ahh, that makes sense, this actually reminds me, if you guys change how things are tagged as "no to be translated" do you also plan to update the corresponding Selenium tests in the monolith that looks for that class on the SIW elements? I just want to make sure we have accounted for those tests as well.

Reply:
Yeah I believe we had this discussion sometime when you were on PTO but @lesterchoi-okta was doing an investigation into the selectors that are used in monolith and whether it will be feasible to convert all of them over to semantic selectors or `data-se`. I'll start a wiki right now to keep track of selectors that we know we've broken or removed as part of the migration

Reply:
As a quick fix for enleaks tests in SIW, I've modified spec by adding `.identifier-container span` in addition to `.no-translate` as selector for text without translation
https://github.com/okta/okta-signin-widget/pull/3487/commits/60e0ccb46256f738c3fc1acbfc4fda9f093b1149#diff-20ecefc4d164e1f22564fa444b02083080727de4a7c205be486bfbacbbbd050d
But it's bad workaround, should remove it.

A better way would be to add class `no-translate` to the outer `Box` (the one with `identifier-container` class). But the problem is that its `.textContent` would include `User` from `<title>User</title>` inside icon.
Any reason why `.textContent` is used instead of `.innerText` ?
https://github.com/okta/okta-signin-widget/blob/b67d788ed46fad0d11c981895969243103fed752/test/testcafe/spec-en-leaks/EnglishLeaks_spec.js#L173

Reply:
I'm not 100% sure how our English leak tests work but I'm assuming we use `.textContent` to also exclude non-visible text that appears in the DOM but not on the page because I believe otherwise our English leak tests will flag non-visible text like `<title>User</title>` as a leak?

TLDR: I think you should be safe to move the `.no-translate` to the outer `Box` for now and then we can remove it once https://github.com/okta/odyssey/pull/2074 is merged in to Odyssey

Reply:
Actually `.textContent` **includes** non-visible text (like content of `title` tag) while `.innerText` does not include.

So if we mix `.textContent`  and `.inerText`
https://github.com/okta/okta-signin-widget/blob/b67d788ed46fad0d11c981895969243103fed752/test/testcafe/spec-en-leaks/EnglishLeaks_spec.js#L164
https://github.com/okta/okta-signin-widget/blob/b67d788ed46fad0d11c981895969243103fed752/test/testcafe/spec-en-leaks/EnglishLeaks_spec.js#L173
it cause problems with hidden text